### PR TITLE
[cli] Add debug statement to sender_can_set_message test

### PR DESCRIPTION
**Pull request by hariria**
## Description

Adding a debug statement for the test on `sender_can_set_message` since this is used for the `hello_blockchain` template and it will help first time Move devs who are unfamiliar with debug / print statements in Move.

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update
- [ ] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [x] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Other (specify)

## Checklist
- [ ] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: aptos-move/move-examples/hello_blockchain/sources/hello_blockchain.move [hunk 1]
```diff
@@ -47,6 +47,11 @@ module hello_blockchain::message {
 
     #[test(account = @0x1)]
     public entry fun sender_can_set_message(account: signer) acquires MessageHolder {
+        use std::debug;
```
Comment (gregnazario) [inline aptos-move/move-examples/hello_blockchain/sources/hello_blockchain.move:50]: Usually you put these outside of the function.  Did you test this?

Comment (hariria) [reply aptos-move/move-examples/hello_blockchain/sources/hello_blockchain.move:50]: Yes i tested it works. For some reason VSCode was complaining about unused imports when I imported at the top of the module so I put it here instead... but I can move it to the top of the module if that's better

Comment (gregnazario) [reply aptos-move/move-examples/hello_blockchain/sources/hello_blockchain.move:50]: Yeah you have to put a #[test_only] annotation, as it's unused outside of tests

Comment (hariria) [reply aptos-move/move-examples/hello_blockchain/sources/hello_blockchain.move]: Done!

**Review by gregnazario — APPROVED** [review]

**Review by igor-p — APPROVED** [review]

**Pull request merged**